app/controllers/passengers_controller.rb

```diff
@@ -0,0 +1,60 @@
+class PassengersController < ApplicationController
+
+  def index
+    @passengers = Passenger.all
+  end
+
+  def show
+    @passenger = find_passenger
+    redirect_to passengers_path and return if @passenger.nil?
+  end
+
+  def create
+    @passenger = Passenger.create(passenger_params)
+    if @passenger.id.nil?
+      render :new
```

Comment:
Just a suggestion for readability?

```suggestion
    unless @passenger.valid?
      render :new, status: :bad_request
```